app/controllers/application_controller.rb

```diff
@@ -5,8 +5,8 @@ class ApplicationController < ActionController::Base
 
   before_action :authenticate_user!
   
-  def after_sign_in_path_for(resource)
-    users_index_path
+  def after_sign_in_path_for(users)
+    users_show_path
```

Comment:
I think we already talked about this convention and fixed it? should be user_show and you can pass in the current user as an argument to get the right path.